products/products.css

```diff
@@ -0,0 +1,158 @@
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css");
+
+:root{
+    --btn-color:#2563eb;
+    --red-color:#d946ef;
+}
+body{
+    margin:0;
+    padding: 0;
+    height: 100vh;
+    box-sizing: border-box;
+    background-color: var(--bg-color);
+}
+
+.app-container{
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-areas: 
+    "header header header header header "
+    "sideBar  p-display p-display p-display p-display"
+    "sideBar  p-display p-display p-display p-display"
+    "sideBar  p-display p-display p-display p-display";
+    
+}
+.product-side-bar{
+    display: flex;
+    flex-direction: column;
+    grid-area: sideBar;
+    box-shadow: 2px 2px 2px 2px #ccc;
+}
+
+.filter-btn{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+
+.rangeslider{
+    width: 75%;
+}
+  
+.myslider {
+    -webkit-appearance: none;
+    background: var(--gray-color);
+    width: 75%;
+    height: 20px;
+    opacity: 2;
+}
+  
+  
+.myslider::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    cursor: pointer;
+    background: var(--red-color)  ;
+    width: 7%;
+    height: 20px;
+}
+  
+  
+.myslider:hover {
+    opacity: 1;
+    cursor: pointer;
+}
+
+.price-range-bar{
+    display: flex;
+    justify-content: space-around;
+}
+
+.side-bar-category{
+    display: flex;
+    flex-direction: column;
+    margin-left: 1.5rem;
+}
+
+.ratings{
+    display: flex;
+    flex-direction: column;
+    margin-left: 1.5rem;
+}
+
+.sort-by{
+    display: flex;
+    flex-direction: column;
+    margin-left: 1.5rem;
+
+}
+
+.bar-heading{
+    font-size: 1rem;
+    font-weight: 600;
+}
+
+#demo{
+    color: #000000;
+}
+
```

Comment:
If this code is not supposed to be in production, would recommend removing it, or using a variable for the color to be consistent.

Reply:
I was using the color for just one time .